ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/LoadDatabase.java

```diff
@@ -158,6 +159,15 @@ private boolean isDbEmpty(String dbName) throws HiveException {
     // Add the checkpoint key to the Database binding it to current dump directory.
     // So, if retry using same dump, we shall skip Database object update.
     parameters.put(ReplUtils.REPL_CHECKPOINT_KEY, dumpDirectory);
+
+    if (needSetIncFlag) {
```

Comment:
No need to check it. Instead just set needSetIncFlag directly using setBoolVar.

Reply:
in case of alter case need not set the flag ..

Reply:
done